### DISCUSSION TITLE
How to use Chart js

### DISCUSSION THREAD

**Discussion by naserianali** [community/community#64934] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to use chart.js ?

**Comment by fedorkanan** [comment]
hi @CyberAli1989  for using chart js you can see the website of library :
https://www.chartjs.org/docs/latest/
and their github to stay updated:
https://github.com/chartjs
and w3school page to learn the hole library :
https://www.w3schools.com/ai/ai_chartjs.asp

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
